log-collector-script/linux/README.md

```diff
@@ -91,7 +91,7 @@ Trying to archive gathered information...
 
 * SSM agent should be installed and running on Worker Node(s). [How to Install SSM Agent link](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-manual-agent-install.html)
 
-* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed role `AmazonSSMManagedInstanceCore` will have all the required permission for SSM agent to run on EC2 instances. The IAM managed role `AmazonSSMManagedInstanceCore` has `S3:PutObject` permission to all S3 resources.
+* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed policy `AmazonSSMManagedInstanceCore` will have all the required permission for SSM agent to run on EC2 instances. Worker Node(s) should have `S3:PutObject` permission to all S3 resources.
```

Comment:
```suggestion
* Worker Node(s) should have required permissions to communicate with SSM service and upload data to your S3 Bucket. The IAM managed policy `AmazonSSMManagedInstanceCore` will have all the required permissions for SSM agent to run on EC2 instances. You will need `S3:PutObject` permission to your S3 resources accordingly.
```

beyond the first sentence is ultimately a suggestion on what policies the user should add. based on the section's intent, i think i'd just update the wording a bit. if it sounds good can apply the same change to both?

Reply:
Thank you for your suggestion.

I modified the wording following it.